tests/kernel/fifo/fifo_timeout/prj.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_ZTEST=y
 CONFIG_IRQ_OFFLOAD=y
+CONFIG_BOOT_DELAY=1000
```

Comment:
@rgundi: Is there any reason for introducing boot delay?

Reply:
Yes. The issue #8159 cannot be reproduced without this delay. It's a very timing dependent issue.

Reply:
you should remove it though, we do not set this for tests by default.